README.md

```diff
@@ -98,9 +98,11 @@ Usage of ./pwru:
       --filter-dst-port uint16      filter destination port
       --filter-func string          filter the kernel functions that can be probed; the filter can be a regular expression (RE2)
       --filter-mark uint32          filter skb mark
+      --filter-netns uint32         filter netns inode
       --filter-proto string         filter L4 protocol (tcp, udp, icmp)
       --filter-src-ip string        filter source IP addr
       --filter-src-port uint16      filter source port
+      --output-limit uint           limit the number of printing, 0 for unlimited (default 1000)
```

Comment:
- Let's make the default 0. Otherwise, the existing users might not notice why it stopped after 1k lines.
- Maybe `--output-stop-after-num-lines` is more clear name?

Reply:
>  Maybe --output-stop-after-num-lines is more clear name?

parameter name should be short and clear. `--output-limit-lines` better?

Reply:
`--output-limit-lines` is better. Could you change the default (to 0), and update the description of the flag to `Exit the program after the number of events has been received/printed`?